src/main/java/org/broadinstitute/hellbender/utils/help/HelpConstants.java

```diff
@@ -8,4 +8,22 @@ public static String forumPost(String post) {
         return GATK_FORUM_URL + post;
     }
 
+    /**
+     * Definition of the group names / categories of tools.
+     * The names get parsed to make supercategories in the doc index,
+     * so be careful when making big changes -- see GATKDoclet.java toMap()
```

Comment:
That's kind of gross...  maybe this should be an enum instead?
Relatedly, these categories are weirdly redundant with but not concordant with ProgramGroups.